PSGSuite/Public/Calendar/New-GSCalendarACL.ps1

```diff
@@ -111,6 +117,9 @@ function New-GSCalendarAcl {
                     Write-Verbose "Inserting new ACL for type '$Type' with value '$Value' in role '$Role'  on calendar '$calId' for user '$U'"
                     $body.Scope = $scopeData
                     $request = $service.Acl.Insert($body, $calId)
+                    if ($DisableNotifications.IsPresent){
+                        $request.sendNotifications = $false
```

Comment:
Hey @Foggy2 - Since a `Switch` parameter always defaults to `$False` and we appear to either be setting it to `$false` if the parameter `IsPresent` here, can we flip this to instead set `$request.SendNotifications` to the inverse of `$DisableNotifications`? Basically, remove the `If` wrapper and replace this line with `$request.sendNotifications = -not $DisableNotifications`. This will allow either scenario to work if someone passes `$DisableNotifications:$false` instead of what's expected. Here's a sample of what I mean:

```powershell
function Test-Switch {
    [CmdletBinding()]
    Param (
        [parameter()]
        [Switch]
        $DisableNotifications
    )
    Process {
        $inverse = -not $DisableNotifications
        Write-Host "SendNotifications is '$inverse'"
    }
}
Test-Switch                                   # This should return True (since it's inverting the default Switch value of False
Test-Switch -DisableNotifications             # This should return False
Test-Switch -DisableNotifications:$false      # This should return True since we're force passing a False value to the Switch (maybe for explicit verbosity/clarity around what that may be doing in a script)
Test-Switch -DisableNotifications:$true       # This should return False
```
![image](https://user-images.githubusercontent.com/12724445/98455799-12f89480-213b-11eb-98ca-f42fcd3f43de.png)